### ISSUE TITLE
nodeSelector usage

### ISSUE THREAD

**Issue by lombardialess** [mongodb/mongodb-kubernetes-operator#1142]
Hello,
for using specific nodepools on the cluster I'm trying to use the nodeSelector but seems not recognize by the operator.
Is it a bug or a feature request?
If it is a feature I think that could be useful to introduce it.
Thanks
Alessandro

**Comment by laurivosandi** [comment]
Hi, in the depths of documentation I managed to find this. For example to get Mongo nodes scheduled on dedicated storage nodes whilst being distributed between AWS AZ-s I used following. I hope this helps:

```
---
apiVersion: mongodbcommunity.mongodb.com/v1
kind: MongoDBCommunity
spec:
  statefulSet:
    spec:
      template:
        spec:
          affinity:
            podAntiAffinity:
              requiredDuringSchedulingIgnoredDuringExecution:
                - labelSelector:
                    matchExpressions:
                      - key: app
                        operator: In
                        values:
                          - mongodb-svc
                  topologyKey: topology.kubernetes.io/zone
          nodeSelector:
            dedicated: storage
          tolerations:
            - key: dedicated
              operator: Equal
              value: storage
              effect: NoSchedule
```

**Comment by irajdeep** [comment]
@lombardialess Feel free to reopen if the sample example provided by @laurivosandi (thanks for your comment) doesn't work for you.

**Comment by dan-mckean** [comment]
Hi, I'm Dan and I'm the Product Manager for MongoDB's support of Kubernetes.

I'm doing some work right now to try and identify how the Community Operator is being used. The Community Operator is something I inherited when I started at MongoDB, but it doesn't get as much attention from us as we'd like and we're trying to understand how it's used in order to establish its future. It will help us prioritize future issues and PRs raised by the community 🙂

Here's a super short survey (it's much easier for us to review all the feedback that way!): [https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link](https://docs.google.com/forms/d/e/1FAIpQLSfwrwyxBSlUyJ6AmC-eYlgW_3JEdfA48SB2i5--_WpiynMW2w/viewform?usp=sf_link)

 If you'd rather email me instead: [dan.mckean@mongodb.com](mailto:dan.mckean@mongodb.com?subject=MongoDB%20Community%20Operator%20feedback)

Thank you in advance!
Dan
